### ISSUE TITLE
Texttop on ARM

### ISSUE THREAD

**Issue by hcjl** [tombh/texttop#12]
Hi Thomas,
I am really excited about Texttop. It would be great supporting ARM. I tried building it on Archlinx armv7l, but I receive an error message. 

Here it is, if you want to investigate. If not, it is an amazing tool anyway;-)

Thanks & Regards,
Jens

```
docker build -t texttop .
Sending build context to Docker daemon 179.7 kB
Step 1 : FROM alpine
latest: Pulling from library/alpine

fae91920dcd4: Pull complete 
Digest: sha256:ea0d1389812f43e474c50155ec4914e1b48792d420820c15cab28c0794034950
Status: Downloaded newer image for alpine:latest
 ---> f70c828098f5
Step 2 : COPY . /app
 ---> a5db764d3a31
Removing intermediate container a42ad72418af
Step 3 : RUN echo "http://mirror1.hs-esslingen.de/pub/Mirrors/alpine/v3.3/main" > /etc/apk/repositories
 ---> Running in 9014dc5c1e50
rpc error: code = 2 desc = "oci runtime error: exec format error"

```

**Comment by tombh** [comment]
Hey Jens! Thanks for the issue. Not having an armv7l chip to hand, it's hard to debug this. My guess is that even the base alpine image needs changing. Could you try changing the `Dockerfile` to use the arm alpine base image? So replace;

`FROM alpine`

with

`FROM resnullius/alpine-armv7l`

**Comment by tombh** [comment]
Related to #4

**Comment by hcjl** [comment]
Hi Thomas,
sorry for the late reply, but I had to work a lot. Unfortunately in the meantime my ARM device died, so I cannot try it anymore. I hope you will keep up this great work.
Thx & Rgds
Jens

**Comment by tombh** [comment]
Ha, shame about the ARM device. But thanks, will definitely keep it
up, it's proving very useful up here in the Himalayas!

**Comment by tombh** [comment]
This should be resolved in #This should be resolved in c415547
